integration_tests/src/lib.rs

```diff
@@ -3624,4 +3758,135 @@ mod test {
 
         client.detach(&volume_id.to_string()).await.unwrap();
     }
+
+    // Test validating a non-block size amount fails
+    #[tokio::test]
+    async fn test_pantry_validate_fail() {
+        const BLOCK_SIZE: usize = 512;
```

Comment:
This seems like a bunch of code for a test that might not need all this?
Could this be simpler in that it has a volume created but without any import,
then just a verify with invalid size?  Like we know it will never get to the
validate part, so why bother actually doing it when the check itself will
stop first?

Reply:
You're right, I simplified this test in 293c712